spinoso-string/src/lib.rs

```diff
@@ -1919,7 +1919,7 @@ impl String {
     /// [binary encoded]: crate::Encoding::Binary
     /// [`str::from_utf8`]: core::str::from_utf8
     #[inline]
-    pub fn codepoints(&self) -> Result<Codepoints<'_>, CodepointsError> {
+    pub fn codepoints(&self) -> Result<Codepoints, CodepointsError> {
```

Comment:
this should ideally not allocate.

Reply:
The b8729b7 version doesn't use `alloc::vec` anymore 👍